### DISCUSSION TITLE
Multi page form - button takes you to bottom of the form

### DISCUSSION THREAD

**Discussion by mattgeminiworktops** [solspace/craft-freeform#234] [Feature Requests]
Created a form using Tailwind Template. The form works fine and submits as it should, however upon clicking 'Next' or 'Previous', the next page of the form loads at the bottom of the screen.

- Craft Pro 3.7.18.2
- Freeform Version: 3.11.12
- Freeform Edition: Pro
- Fresh Install or Upgrade: Fresh Install

Loom of the form in action: https://www.loom.com/share/2837020704db467b9f9f9ce8bbd4a345

**Comment by kjmartens** [comment]
Do you have the **Automatically Scroll to top of the Form on AJAX submit?** setting enabled for your site?

**Comment by mattgeminiworktops** [comment]
Yes I do.
I am using Tailwind.

**Comment by kjmartens** [comment]
Sorry for the delay. I tried this again using Tailwind template (not that it should make a difference), and it still works fine for me. I realize the issue now though. The _Automatically Scroll to top of the Form on AJAX submit?_ setting I am referring to is a new feature for Freeform 3.12+ only. It's currently in beta, but we anticipate releasing it today. Whenever you're ready to upgrade to Freeform 3.12, just toggle that new setting on (under "Form Behavior" settings page) and it'll work as you wish. 🙂
